readme.md

```diff
@@ -1,139 +1,99 @@
-# RavenDB - An ACID NoSQL Document Database
-This repository contains source code for the [RavenDB](https://ravendb.net/) document database. With a RavenDB database you can set up a NoSQL data architecture or add a NoSQL layer to your current relational database.
+<h1 align="center"><img src="docs/Server guide banner.png" alt="Ory Keto - Open Source & Cloud Native Access Control Server">RavenDB - Modern ACID NoSQL Database Solution</h1>
 
-![RavenDB Studio](docs/readmeScreenshot.png)
 
-## Supported Platforms
-- Windows
-- Linux
-- Docker
-- MacOS
-- Raspberry Pi
+<h4 align="center">    
+    <a href="https://discord.com/invite/ravendb">Discord 🌐💬🎙️</a> |
+    <a href="https://github.com/ravendb/ravendb/discussions">Discusssions 🎫</a> |
+    <a href="https://www.youtube.com/@ravendb_net">YouTube ▶️</a> 
+    <br/><br/>
+    <a href="live-test.ravendb.net/">🔌 Try from your browser, now! 🔌</a>
+    <br/><br/>
+    <a href="https://ravendb.net/docs">🎒👶 Documentation</a> |
+    <a htef="https://ravendb.net/articles/">📰 Articles</a> | 
+    <a htef="https://ravendb.net/book/"> Book 📖🍵 </a>
+    <br/><br/>
+    <a href="https://ravendb.net/buy#developer">💹💰 Free developer license </a>
+    <br/><br/>
+</h4>
 
-## Grab Your License and Download Latest Version
 
-Request [your license](https://ravendb.net/free).
 
-Download [the latest version of RavenDB](https://ravendb.net/downloads).
+[RavenDB](https://ravendb.net/) is a NoSQL database forged in the flames of passion, by the people frustrated with the state of database industry. Fusing extreme performance and tranquility of ease-of-use, RavenDB offers **above-the-roof developer experience**. While we take the heavy lifting of managing your data (and a lot more..) on our shoulders, you can focus on your system and application code.
 
-## Getting Started
-Install and [set up your database](https://ravendb.net/docs/article-page/7.0/csharp/start/getting-started).
+## We're
 
-## Learn RavenDB Quickly
-[RavenDB Bootcamp](https://ravendb.net/learn/bootcamp) is a free, self-directed learning course. In just three units you will learn how to use RavenDB to create fully-functional, real-world programs with NoSQL Databases. If you are unfamiliar with NoSQL, it’s okay. We will provide you with all the information you need.
+- Completely open-source
 
-## Stay Updated on New Developments
-We keep adding new features to improve your RavenDB experience. Check out [our latest improvements](https://ravendb.net/docs/article-page/7.0/csharp/start/whats-new), updated weekly.
+- Fully ACID & Safe by default
```

Comment:
Safe or Secure? Asking since in other places we go with "Secure by default" mantra

Reply:
secure - thanks